remove-user-on-logout/src/main/java/de/intension/rest/LicenseConnectRestClient.java

```diff
@@ -0,0 +1,80 @@
+package de.intension.rest;
+
+import static org.jboss.logging.Logger.getLogger;
+
+import java.io.Closeable;
+import java.io.IOException;
+
+import org.apache.http.HttpHeaders;
+import org.apache.http.client.config.RequestConfig;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.jboss.logging.Logger;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import de.intension.rest.model.RemoveLicenseRequest;
+
+public class LicenseConnectRestClient
```

Comment:
Thinking if it would make sense to have one commmon licensecollect restclient. able to remove and add licenses...
BUt I think classes are different enough and so it is only the auth in common.
You can leave it this way.